pulp_ansible/app/custom_fields.py

```diff
@@ -16,7 +16,7 @@
     get_groups_with_perms_attached_roles,
     get_perms_for_model
 )
-from django.contrib.auth.models import Group
+from pulpcore.plugin.models import Group
```

Comment:
I guess, this should have been the proxy model for a long time... Thanks for finding it.